CHANGELOG

```diff
@@ -4,6 +4,13 @@
 
 https://openslides.org/
 
+Version 2.1 (unreleased)
+========================
+
+Core:
+- Use channels instead of tornado.
+- Therefore support big assemblies, with lots of users.
+
 Version 2.0.1 (unreleased)
```

Comment:
You can merge the list for 2.0.1 into the new list for 2.1 because we won't release a 2.0.1 anymore if the channels thing is in master.